### ISSUE TITLE
draw:drawvertex is not triggered

### ISSUE THREAD

**Issue by devdattaT** [Leaflet/Leaflet.draw#552]
I want to get the event whenever a new vertext is created while the user is drawing a Line. The `draw:drawvertex` Event seems to be appropriate for this, but it does not seem to be triggered, ever.

I tried wring it up like this:

```
map.on('draw:drawvertex', function (e) {
    console.dir(e);
});
```
But it is never fired.

Here is a reproducible example: 
https://jsfiddle.net/Devdattat/sabbx2d9/

**Comment by SirenHound** [comment]
My guess is that you seem to be using version 0.2.3 and that event was added in 0.2.4-dev

**Comment by devdattaT** [comment]
@SirenHound You were right. This functionality is available only in the `0.3.0` or the `0.3.0-dev` versions of the Leaflet.draw Library.
